src/workerd/api/modules.h

```diff
@@ -10,13 +10,26 @@
 #include <workerd/api/sockets.h>
 #include <workerd/api/unsafe.h>
 #include <workerd/api/worker-rpc.h>
+#include <workerd/io/features.h>
 #include <workerd/io/worker.h>
 #include <workerd/jsg/modules-new.h>
 
 #include <cloudflare/cloudflare.capnp.h>
 
 namespace workerd::api {
 
+class EnvModule final: public jsg::Object {
+ public:
+  kj::Maybe<jsg::JsObject> getCurrent(jsg::Lock& js);
+
+  jsg::JsValue withEnv(jsg::Lock& js, jsg::Value newEnv, jsg::Function<jsg::JsValue()> fn);
```

Comment:
Can you add a comment to both of these functions and explain what they do? It's not really visible the difference between getCurrent and withEnv by looking at the function names